src/ir/ty.rs

```diff
@@ -901,6 +901,13 @@ impl Type {
             _ => {}
         }
 
+        if location.kind() == CXCursor_ClassTemplatePartialSpecialization {
+            // Sorry! (Not sorry)
+            warn!("Found a partial template specialization; bindgen does not \
```

Comment:
The last commit regarding partial template specializations obviates the need for this.